db/sql_changeset_schema

```diff
@@ -39,6 +39,34 @@ CREATE INDEX user_account_devices_userid_idx on user_account_devices(userid);
 CREATE INDEX user_files_userid_idx on user_files(userid);
 CREATE INDEX user_files_deviceid_idx on user_files(deviceid);
 
+-- share file
+CREATE TABLE user_share_files (
+    id SERIAL PRIMARY KEY,
+    ownerid INTEGER NOT NULL,
+    uuid TEXT UNIQUE,
+    filepath TEXT NOT NULL,
+    name TEXT NOT NULL,
+    type TEXT NOT NULL,
+    public_access BOOLEAN NOT NULL
+);
+
+CREATE TABLE user_share_files_access (
+    id SERIAL PRIMARY KEY,

```

Comment:
id is not a primary key at all for access. 
Combination (uuid or fileid, userid) - is a primary key